pkgs/applications/science/logic/isabelle/components/isabelle-linter.nix

```diff
@@ -2,16 +2,26 @@
 
 stdenv.mkDerivation rec {
   pname = "isabelle-linter";
-  version = "Isabelle2021-1-v1.0.0";
+  version = "Isabelle2022";
```

Comment:
This is nowhere taggged or the commit contains a version bump so this must use unstable-XXXX-XX-XX.